gridt/src/app/about/privacy/privacy.page.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { ModalController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-privacy',
+  templateUrl: './privacy.page.html',
+  styleUrls: ['./privacy.page.scss'],
+})
+export class PrivacyPage implements OnInit {
+
+  constructor(public modalController: ModalController) { 
+
```

Comment:
Nit picking: In these cases I normally do `constructor () {}` instead of the redundant line breaks.